src/containers/portal/utils.js

```diff
@@ -8,7 +8,7 @@ const CONSTITUTION_HASH = 'QmcHB9GKHAKCLQhmSj71qNJhENJJg8Gymd1PvvsCQBhG7M';
 
 // test root
 // const CONTRACT_ADDRESS_GIFT =
-//   'bostrom1kytv5yjkkszr2zs8937lp6kvqhxxx2e429qwqk2dr7rh99pklf9scxlrwx';
+//   'bostrom1dwzfa74hzpt6393czajlldnxjup8zk3xh3skegnm67yzqx33k2cssyduk8';
```

Comment:
this correct?

Reply:
Yes , new dev contracts

Reply:
but in production should be old contract?

Reply:
in production use prod contact